### DISCUSSION TITLE
Tox 3.26.0 release breaking builds as toml dependency changed to tomli

### DISCUSSION THREAD

**Discussion by deepika094** [tox-dev/tox#2494] [Q&A] [answered]
Hi,

RE: Use ``tomllib`` on Python 3.11 or later and ``tomli`` instead of ``toml`` library on lower versions - by :user:`hroncok`

This is breaking lots of our builds since we were importing toml and that was coming from tox earlier and not tox is downloading tomli instead of toml.. We will have to explicitly add toml dependency in all the builds in order to fix this.. 
Was this intentional?

**Comment by gaborbernat** [comment, marked as answer]
> This is breaking lots of our builds since we were importing toml and that was coming from tox earlier and not tox is downloading tomli instead of toml.

If you're depending on a package you really should declare as a dependecy to your project and not really on other projects pulling it in for you. Totally intentional.
